async def raise_exception():
    raise Exception()


await raise_exception()
